src/main/Elevator.java

```diff
@@ -206,6 +217,12 @@ public void run() {
 		            tempElevator.setDoorsOpen(false);
 		        }
 		    };
+		    try {
+				Thread.sleep(Configurations.TIME_TO_LOAD_UNLOAD);
+			} catch (InterruptedException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
 		    Timer timer = new Timer("Timer");
 		    long delay = fe == null ? 500 : 
 		    	fe.getErrorCode() == 1 ? 1000 : 500;
```

Comment:
The delay is basically how long the door timer is, basically change the 500 to Configurations.Time_To_Unload